### PR TITLE
Debian packaging

### PR DESCRIPTION
**Pull request by anephtali**

### CODE REVIEW

File: debian/beamium.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=beamium
+After=network.target
+
+[Service]
+EnvironmentFile=-/etc/default/beamium
+ExecStart=/bin/bash -c "if [ \"x$START_ON_INSTALL\" = \"xtrue\" -o \"x$START_ON_INSTALL\" = \"xyes\" -o \"x$START_ON_INSTALL\" = \"x0\" ] ; then /usr/bin/beamium --config=/etc/beamium/config.yaml; else /bin/echo \"beamium not configured to start, please edit /etc/default/beamium to enable\"; fi"
```
Comment (tgermain) [inline debian/beamium.service]: This make the program appear in journald logs as bash.

You can get a better behavior by using `ExecPreStart` directive :
```
ExecStartPre=/bin/bash -c "if [ \"x$START_ON_INSTALL\" = \"xtrue\" -o \"x$START_ON_INSTALL\" = \"xyes\" -o \"x$START_ON_INSTALL\" = \"x0\" ] ; then exit 0; else /bin/echo \"beamium not configured to start, please edit /etc/default/beamium to enable\"; fi"
ExecStart=/usr/bin/beamium --config=/etc/beamium/config.yaml
```

That way you still have your check on start_on_install and have the correct program in journald

**Comment by arnaudmorin** [comment]
can we merge that?

File: debian/beamium.service (hunk 1 above)
Comment (anephtali) [reply debian/beamium.service]: I will check that this week-end and make the necessary changes.

**Comment by anephtali** [comment]
Commits will be squashed once reviewed.

**Comment by arnaudmorin** [comment]
Just a though: why do we continue to install that in /opt?

**Comment by anephtali** [comment]
I kept the old package behaviour to avoid breaking external softwares that check the content of this directory.

**Comment by StevenLeRoux** [comment]
For more conformity we the package could install binary into /usr/sbin, config into /etc/beamium and keeping /opt/beamium for data partitioning purpose.

**Comment by anephtali** [comment]
I moved the binary into /usr/sbin.

**Comment by tgermain** [comment]
Any update on that ?

**Comment by d33d33** [comment]
Thanks @anephtali

**Pull request merged**